index.js

```diff
@@ -5,3 +5,8 @@ todoActions();
 
 const todoClass = new Crud();
 todoClass.updateUI();
+
+// eslint-disable-next-line import/prefer-default-export
+export function sum(a, b) {
+  return a + b;
+}
```

Comment:
- [optional] Please I think you can resolve the import rules instead of disabling the linter. More so, it is like the portion of code is redundant; if so, kindly remove it to make your codebase cleaner. 👍